conans/test/integration/toolchains/gnu/test_pkgconfigdeps.py

```diff
@@ -756,6 +758,7 @@ def build_requirements(self):
             def generate(self):
                 tc = PkgConfigDeps(self)
                 tc.build_context_activated = ["tool"]
+                tc.build_context_suffix = {"tool": ""}  # No suffix defined
```

Comment:
Better not modify existing tests, add new

Reply:
I think this should still be not modified and unnecessary. The validation needs to check that the new folder is defined, and validate otherwise